Python/ceval.c

```diff
@@ -722,21 +724,23 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
         /* Because this avoids the RESUME,
          * we need to update instrumentation */
         _Py_Instrument(_PyFrame_GetCode(frame), tstate->interp);
-        monitor_throw(tstate, frame, frame->prev_instr);
+        monitor_throw(tstate, frame, frame->instr_ptr - 1);
```

Comment:
Why the `- 1`?
We are throwing into a paused generator here, so `instr_ptr` points to the same `YIELD_VALUE` as `prev_instr` as did. Doesn't it?